### PR TITLE
release: change version back to .dev after release

### PR DESCRIPTION
**Pull request by dimaqq**
change version to dev flavour of the expected upcoming release, `2.15.0.dev0`

### CODE REVIEW

**Review by benhoyt — APPROVED** [review]

File: ops/version.py [hunk 1]
```diff
@@ -17,4 +17,4 @@
 This module is NOT to be used when developing charms using ops.
 """
 
-version: str = '2.14.0'
+version: str = '2.14.0.dev0'
```
Comment (tonyandrewmeyer) [inline ops/version.py:20]: We're working towards 2.15 now.

```suggestion
version: str = '2.15.0.dev0'
```

**Review by tonyandrewmeyer — CHANGES_REQUESTED** [review]

**Comment by tonyandrewmeyer** [comment]
We should probably standardise and document the conventional commit type for these changes. `release` isn't one we've used before as far as I know, and it's not in the usual list of types (I'd probably go for `chore`). We could adopt it, but we should be consistent about it.

**Comment by dimaqq** [comment]
Thanks, Tony. The dreaded off-by-one error 🙈

**Review by tonyandrewmeyer — APPROVED** [review]

**Comment by dimaqq** [comment]
> We should probably standardise and document the conventional commit type for these changes. `release` isn't one we've used before as far as I know, and it's not in the usual list of types (I'd probably go for `chore`). We could adopt it, but we should be consistent about it.

`chore(release): waffle waffle` then?

**Pull request merged**